aiohttp/client_exceptions.py

```diff
@@ -7,13 +7,17 @@
 
 from .typedefs import StrOrURL
 
-try:
+if TYPE_CHECKING:
     import ssl
 
     SSLContext = ssl.SSLContext
-except ImportError:  # pragma: no cover
-    ssl = SSLContext = None  # type: ignore[assignment]
+else:
+    try:
+        import ssl
 
+        SSLContext = ssl.SSLContext
+    except ImportError:
```

Comment:
Not sure if you missed my other comment, but still a coverage regression:
https://github.com/aio-libs/aiohttp/pull/10099#issuecomment-2520352679

Reply:
I did (`# pragma: no cover`), Wouldn't it be better to remove the try/except block?

Reply:
Well, yes, that's what we suggested earlier. We don't the the TYPE_CHECKING either then. asvetlov is keen to keep it for now though (https://github.com/aio-libs/aiohttp/pull/10099#issuecomment-2517971410). Though I can't imagine a anyone using a version of Python without SSL and wanting to use aiohttp on it..